spec/lib/rex/proto/ms_dtyp/ms_dtyp_ace_spec.rb

```diff
@@ -0,0 +1,313 @@
+RSpec.describe Rex::Proto::MsDtyp::MsDtypAce do
+  let (:domain_sid) { "S-1-5-21-#{rand(0xf00000..0xffffffff)}-#{rand(0xf00000..0xffffffff)}-#{rand(0xf00000..0xffffffff)}-#{rand(100..999)}" }
+
+  describe '.from_sddl_text' do
+    it 'raises an exception on invalid ACEs' do
+      expect { described_class.send(:from_sddl_text, '', domain_sid: domain_sid) }.to raise_error(RuntimeError)
```

Comment:
It'd be great to be a bit more rigorous in what we expect here, so unexpected exceptions aren't caught accidentally

https://docs.rubocop.org/rubocop-rspec/cops_rspec.html#rspecunspecifiedexception